content/en/docs/concepts/overview/components.md

```diff
@@ -53,8 +53,8 @@ Some types of these controllers are:
   * Node controller: Responsible for noticing and responding when nodes go down.
   * Job controller: Watches for Job objects that represent one-off tasks, then creates
     Pods to run those tasks to completion.
-  * Endpoints controller: Populates the Endpoints object (that is, joins Services & Pods).
```

Comment:
this is still runs, there is also a `EndpointSliceMirroring` controller that mirror the Endpoints with the EndpointSlices to have full compatibility

Reply:
The aim of the commit that this text comes from is to make EndpointSlice _more prominent_. Removing this example is part of that.